client/src/controllers/KeyboardController.ts

```diff
@@ -0,0 +1,23 @@
+import { Controller } from '@hotwired/stimulus';
+import Mousetrap from 'mousetrap';
+
+export class KeyboardController extends Controller<HTMLButtonElement> {
+  static values = { key: String };
```

Comment:
```suggestion
  static values = { key: { default: '', type: String } };
```

Better to be explicit about the default if we can.